script.js

```diff
@@ -1,4 +1,161 @@
+
+//Create a version of the Beat That dice game, where players create the largest number 
+//they can by concatenating random dice roll digits
+//There are 2 players and players take turns.
+//When a player clicks Submit, the game rolls 2 dice and shows the dice rolls, for example 3 and 6.
+//The player picks the order of the dice they want. 
+//For example, if they wanted the number 63, they would specify that the 2nd dice goes first. 
+//You can choose how the player specifies dice order.
+//After both players have rolled and chosen dice order, the player with the higher combined number wins.
+
+var GAME_MODE_DICE_ROLL = 'dice roll';
+var GAME_MODE_DICE_ORDER = 'dice order';
```

Comment:
Good job naming constants in UPPER_CAMEL_CASE :) 1 more thing is the string should also be exactly the same as the constant's name.